File: TenMinsJava/day19.java
// output a number, and wait for a keypress, then increment the number and print it out, then repeat
package TenMinsJava;

import java.util.Scanner;

public class day19 {
    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
        int num = 0;
        System.out.println("Enter to increment and q to quit");
        while (true) { 
            if (scn.nextLine().toLowerCase().equals("q")) {
                break;
            }
            else {
                num += 1;
                System.out.print(num);
            }
        }
    }
}

File: TenMinsPy/day19.py
# output a number, and wait for a keypress, then increment the number and print it out, then repeat
import keyboard
class day19:
    def __init__(self):
        num = 0
        while True:
            print(num)
            keyboard.read_event()
            num += 1
            if num == 10:
                break

day19()